variables.tf

```diff
@@ -38,6 +38,43 @@ variable "address_allocation_ids" {
   default     = []
 }
 
+variable "security_group_enabled" {
+  type        = bool
+  description = "Whether to create default Security Group for AWS Transfer Server."
+  default     = true
+}
+
+variable "security_group_description" {
+  type        = string
+  default     = "AWS Transfer Server Security Group"
+  description = "The Security Group description."
+}
+
+variable "security_group_use_name_prefix" {
+  type        = bool
+  default     = false
+  description = "Whether to create a default Security Group with unique name beginning with the normalized prefix."
+}
+
+variable "security_group_rules" {
+  type = list(any)
+  default = [
+    {
+      type        = "ingress"
+      from_port   = 22
+      to_port     = 22
+      protocol    = "tcp"
+      cidr_blocks = ["10.0.0.0/16"]
```

Comment:
should this be an empty array by default?

Reply:
@Benbentwo I copied the pattern from [this repo](https://github.com/cloudposse/terraform-aws-elasticache-redis/blob/master/variables.tf#L30-L47).

It also has the `cidr_block` array populated with `0.0.0.0/0` but I don't like having it open to the world so leaving `cidr_block` with `[]` will show an error explaining it needs to be set. Not the greatest UX unfortunately but I think security should take precedence over UX.